src/ImageList/manifest.json

```diff
@@ -39,6 +39,25 @@
     }
   ],
   "childComponents": [
+    {
```

Comment:
I just want to double check that this isn't in the wrong place or anything. Is this new?